ccdproc.cfg

```diff
@@ -1,5 +0,0 @@
-[ccdproc]
-# auto_logging = True
-# Whether to automatically log operations to metadata
-# If set to False, there is no need to specify add_keyword=False
-# when calling processing operations
```

Comment:
As ccdproc has some custom config items, this should be documented using the sphinx extension: https://github.com/astropy/sphinx-astropy/blob/main/sphinx_astropy/ext/generate_config.py

Reply:
Good call, @saimn . Added a commit. How does it look now?

Reply:
Looks good :+1: